apps/condo/lang/en.json

```diff
@@ -255,8 +255,14 @@
   "pages.condo.error.Description": "We are carrying out technical work. Please come back a little later.",
   "pages.condo.contact.PageTitle": "Contacts",
   "pages.condo.contact.ImportTitle": "contacts",
-  "pages.condo.property.EmptyBuildingDescription": "Fill required inputs and we will visualize it",
-  "pages.condo.property.EmptyBuildingHeader": "Describe the house",
+  "pages.condo.property.EmptyBuildingBlock.view.EmptyBuildingDescription": "Map creation may be done by two ways:\nmanually by filling some property parameters or ",
```

Comment:
`Map creation can be done in two ways:\nmanually by filling some property parameters or ` (and second way description is missing)

Reply:
@akarjakin second way is link button, so I don't know how to pass it value to one intl string